app/mqtt_writer.py

```diff
@@ -0,0 +1,37 @@
+"""MQTT Writer module.
+
+This module provides a client class for publishing messages to MQTT brokers.
+
+"""
+
+import logging
+
+import paho.mqtt.client as mqtt
+
+from app.message import ErrorMessage
+from app.mqtt_client import MqttClient
+
+
+class MqttWriter(MqttClient):
```

Comment:
same thing as above. I like the idea to keep them separated. I would strongly recommend to use composition over inheritance though